_beats/libbeat/scripts/generate_fields_docs.py

```diff
@@ -72,10 +72,12 @@ def document_field(output, field, field_path):
         if not field["enabled"]:
             output.write("{}\n\n".format("Object is not enabled."))
 
+    output.write("--\n\n")
```

Comment:
The diff with https://raw.githubusercontent.com/elastic/beats/7.2/libbeat/scripts/generate_fields_docs.py includes a bunch of other changes.

Reply:
Do you have a mechanism to sync that file to this project?

Reply:
yeah, I support shortcutting it though since this branch should only take documentation updates.  @bmorelli25 is on it.

Reply:
:heart: